core/src/apps/webauthn/fido2.py

```diff
@@ -381,23 +368,25 @@ async def read_cmd(iface: io.HID) -> Cmd | None:
         data = ifrm.data
         datalen = len(data)
         seq = 0
+        cid = ifrm.cid  # local_cache_attribute
```

Comment:
simplification logic broke this -- please rename to `ifrm_cid` and there is another `cfrm_cid` a couple dozen lines below

Reply:
Done in [2f64875](https://github.com/trezor/trezor-firmware/pull/2525/commits/2f64875fd31a9f1ad8993bab35363c2ec0df3db4)